config/berichtencentrum-sync-with-kalliope/config.json

```diff
@@ -37,6 +37,7 @@
     "<https://data.vlaanderen.be/id/concept/BesluitDocumentType/95a6c5a1-05af-4d48-b2ef-5ebb1e58783b>",
     "<https://data.vlaanderen.be/id/concept/BesluitDocumentType/6d1a3aea-6773-4e10-924d-38be596c5e2e>",
     "<https://data.vlaanderen.be/id/concept/BesluitType/41a09f6c-7964-4777-8375-437ef61ed946>",
-    "<https://data.vlaanderen.be/id/concept/BesluitType/79414af4-4f57-4ca3-aaa4-f8f1e015e71c>"
+    "<https://data.vlaanderen.be/id/concept/BesluitType/79414af4-4f57-4ca3-aaa4-f8f1e015e71c>",
+    "<https://data.vlaanderen.be/id/concept/BesluitDocumentType/cc831628-95a0-4874-bad5-cdf563896032>"
```

Comment:
Not sure about this.

Reply:
Did they mention something about this form needing to sync with kalliope?

Reply:
No I just assumed it does but I'm not familiar with that, I'll ask.

Reply:
Got the reply, it does need a kalliope sync.